### PR TITLE
WIP: Update Changelog Contributors and 1.7.5 version

### PR DESCRIPTION
**Pull request by jolelievre**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.5.x
| Description?  | Update Changelog Contributors and 1.7.5 version
| Type?         | improvement
| Category?     | CO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | 
| How to test?  | 

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/11089)

### CODE REVIEW

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -413,6 +434,7 @@ GitHub contributors:
 - moncef102
 - montes
 - mplh
+- Mreker
```
Comment (MathiasReker) [inline CONTRIBUTORS.md]: Please change to: Mathias Reker

Comment (PierreRambaud) [reply CONTRIBUTORS.md]: I think this file is automatically generated depending and what you add in your git config.

Comment (eternoendless) [reply CONTRIBUTORS.md]: This information comes from your commit messages @MathiasReker. If you want to be credited differently, consider changing your git configuration.

File: CONTRIBUTORS.md [hunk 2]
```diff
@@ -260,8 +271,10 @@ GitHub contributors:
 - iNem0o
 - Inetbiz
 - iqit-commerce
+- iqit-commerce (Marcin Sz)
```
Comment (eternoendless) [inline CONTRIBUTORS.md]: This is a duplicate

File: CONTRIBUTORS.md [hunk 3]
```diff
@@ -75,8 +80,10 @@ GitHub contributors:
 - bumbu
 - Burhan
 - Caleydon Media
+- can
 - cam.lafit
 - Captain FLAM
+- Captain Youz
```
Comment (eternoendless) [inline CONTRIBUTORS.md]: Duplicate of Anas Mammeri

File: docs/docker/prestashopCli.sh [hunk 4]
```diff
@@ -26,9 +26,9 @@ function usage {
     echo ""
     echo "$0 -c "
     echo ""
-    echo "Build version 1.7.4.0 from branch 1.7.4.x and deploy it to $CURRENT_DIR/nginx_fpm_supervisord/prestashop"
+    echo "Build version 1.7.5.0 from branch 1.7.5.x and deploy it to $CURRENT_DIR/nginx_fpm_supervisord/prestashop"
     echo ""
-    echo "$0 -b 1.7.4.x -v 1.7.4.0 -d nginx_fpm_supervisord"
+    echo "$0 -b 1.7.5.x -v 1.7.5.0 -d nginx_fpm_supervisord"
```
Comment (eternoendless) [inline docs/docker/prestashopCli.sh]: This file should be changed in a different PR

**Review by eternoendless — CHANGES_REQUESTED** [review]

File: CONTRIBUTORS.md (hunk 1 above)
Comment (MathiasReker) [reply CONTRIBUTORS.md]: I fixed my git config :)

Comment (jolelievre) [reply CONTRIBUTORS.md]: indeed the lidt is generated automatically and then injected manually in the file, I am gonna fix the duplicate for the next build

File: docs/docker/prestashopCli.sh (hunk 4 above)
Comment (jolelievre) [reply docs/docker/prestashopCli.sh]: ok

**Comment by mickaelandrieu** [comment]
Hello @jolelievre,

can we close this one?

Sounds like we don't need it anymore :+1:

**Comment by jolelievre** [comment]
yes we can remove it, it's an old one!
